pages/index.tsx

```diff
@@ -103,8 +105,8 @@ export async function getServerSideProps() {
     return {
       id: monitor.id,
       name: monitor.name,
-      tooltip: monitor?.tooltip,
-      statusPageLink: monitor?.statusPageLink
+      tooltip: 'Default Tooltip Value', //monitor?.tooltip,
+      statusPageLink: 'statusPageLink', //monitor?.statusPageLink
```

Comment:
Our program still needs the value in monitor.tooltip, we just shouldn't fail to compile when it's missing. The best way to do this is to declare the full TS type for the configuration. But I might prefer to be lazy and just add `// @ts-ignore`.